index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Solitaire</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.6.1/lodash.min.js" defer></script>
+    <script src="js/solitaire.js" defer></script>
+    <script src="js/deckContainer.js" defer></script>
+
+    <script src="js/table.js" defer></script>
+    <script src="js/app.js" defer></script>
+  </head>
+  <body>
+
+    <div class="table_view" data-component="table">
+
+      <div class="deck_view" data-component="deckContainer"></div>
+      <div class="opened_cards_view" data-component="openedCardsContainer"></div>
+      <div class="houses_view" data-component="housesContainer"></div>
+      <div class="cells_view" data-component="cellsContainer"></div>
+
+    </div>
+
+
+  <script type="text/template" id="deck_template">
+    <% deckCards.forEach(function (card) { %>
+      <div class="card closed"></div>
+    <% }); %>
+  </script>
```

Comment:
I placed templates in html according to first Misha's example, before he showed us WebPack and Handlebars.